src/main/java/mate.academy/lib/Injector.java

```diff
@@ -8,6 +20,45 @@ public static Injector getInjector() {
     }
 
     public Object getInstance(Class<?> interfaceClazz) {
-        return null;
+        Class<?> clazz = findImplementation(interfaceClazz);
+        Object clazzImplementationInstance = createNewInstance(clazz);
+        for (Field field : clazz.getDeclaredFields()) {
+            if (field.isAnnotationPresent(Inject.class)) {
+                Object fieldInstance = createNewInstance(findImplementation(field.getType()));
+                field.setAccessible(true);
+                try {
+                    field.set(clazzImplementationInstance, fieldInstance);
+                } catch (IllegalAccessException e) {
+                    throw new RuntimeException(e);
+                }
+            }
+        }
+        return clazzImplementationInstance;
```

Comment:
The method getInstance does not check for annotations before creating a new instance. Ensure that the class has the necessary annotations before attempting to create an instance.